docs/apis/table.md

```diff
@@ -536,6 +536,29 @@ Table result = left.unionAll(right);
     </tr>
 
     <tr>
+      <td><strong>Minus</strong></td>
+      <td>
+        <p>Similar to a SQL EXCEPT clause. Except returns records from the first table that do not exist in the second table. Duplicate records in the first table are returned exactly once, i.e., duplicates are removed. Both tables must have identical schema, i.e., field names and types.</p>
```

Comment:
"Except returns records" should be "Minus returns records" to be consistent. I would also use "left/right" table instead of first and second according to your example code.